cal/cal/static/js/components/calendar.js

```diff
@@ -108,6 +108,16 @@ analyticsApp.controller('CalendarCtrl', function ($scope, $http, $q, uiCalendarC
      return element;
    };
    
+   $scope.loading = function(isLoading, view){
```

Comment:
what about like `$scope.eventRender` right above it? or are we going to refactor those things later? changing this anyway, just wondering